javascript-typing-tutor/main.js

```diff
@@ -5,11 +5,11 @@ var charsArr = Array.from($chars);
 var currentIndex = 0;
 
 document.addEventListener('keydown', function (event) {
-  charsArr[currentIndex + 1].className = 'underscore';
   if (charsArr[currentIndex].textContent === event.key) {
     charsArr[currentIndex].className = 'correct';
     currentIndex++;
+    charsArr[currentIndex].className = 'underscore';
   } else {
-    charsArr[currentIndex].className = 'wrong';
+    charsArr[currentIndex].classList.add('wrong');
```

Comment:
You should stick to just using `className` or `classList`, not both in the same function. Consistency is key!